### PR TITLE
Added Auto Compare

### PR DESCRIPTION
**Pull request by solomonmcharo28**

### CODE REVIEW

File: CDC-Data-Reconciliation-Backend/auto-compare.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+# Load config.json
+app.dir = os.path.dirname(__file__)
+
+
+config_file_path = os.path.join(app.dir, "config.json")
+with open(config_file_path, "r") as f:
+    app.config = json.load(f)
+
+def main():
```
Comment (Carson21) [inline CDC-Data-Reconciliation-Backend/auto-compare.py:34]: The main method is never called... Put 
`if __name__ == "__main__": 
    main()
`
at the bottom of your code

**Review by Carson21 — COMMENTED** [review]
Please test your code before making a pull request

**Review by Carson21 — APPROVED** [review]
Looks good

**Comment by john-baute** [comment]
Looks good to me too

**Pull request merged**